code/CO2_monitor.ino

```diff
@@ -174,6 +201,9 @@ void setup(){
   pinMode(WARNING_DIODE_PIN, OUTPUT);
 
 #if WIFI_ENABLED
+  #if WIFI_MQTT
+  client.setServer(mqttserver, 1883);
```

Comment:
Following my comment/suggestion from above this might need to be changed to
`mqttClient.setServer(mqttserver, 1883);`